gulp/tasks/svg.js

```diff
@@ -57,7 +57,7 @@ gulp.task('svg:sprite', () => {
     }
 
     const sprites = getSprites();
-    return sprites.map(sprite => bundle(sprite));
+    return sprites.forEach(sprite => bundle(sprite));
```

Comment:
Good idea. I don’t have that much experience with streams, but I’d give it a try.
Another suggestion: when you use forEach or similar higher order functions, you can do just
```js
sprites.forEach(bundle);
```